definitions/variable/tag_transport_types.yaml

```diff
@@ -17,8 +17,8 @@
   - Maritime|Hydrogen:
       description: maritime hydrogen-based transportation technologies
 
-  - Rail:
-      description: rail-based transportation technologies
+#  - Rail:
```

Comment:
Commenting out sections of code is usually not a good idea...

Reply:
This is related with the previous points. This variable was commented to avoid duplication, but if I removed it directly it would appear that it was missing by mistake from the Transport tag.

Just let me know how would you prefer to proceed about it.

Reply:
This connects to my comment about the commented out test case. `- Rail` should be safe to remove.

Reply:
handled in commit https://github.com/openENTRANCE/openentrance/pull/165/commits/a76334dc36d9fa33bb4f9f5dcb36f673daf8318a